css/IE7.css

```diff
@@ -0,0 +1,31 @@
+#main-menu {
```

Comment:
Please remove it or specify more straigth selectors. Importants should not be used, and we do not need to support ie7 any more.